app/backend/src/api/controllers/SyncStatusController.ts

```diff
@@ -11,32 +11,15 @@ import {
   sendSuccessResponse,
 } from "@/api/utils/responseUtils";
 import { chainTableMapping } from "@/database/repositories/BlockValueRepository";
-import whitelisted from "@/core/clients/coingecko/assets/whitelisted.json";
-import { ethers } from "ethers";
-
-interface AssetMap {
-  [chainId: number]: {
-    [address: string]: string;
-  };
-}
+import { whitelistedMap } from "@/core/clients/coingecko/assets/types";
```

Comment:
Alternatively, instance can be constructed in Application.ts or some other root class and passed into the controller constructor.